client/src/components/Form.js

```diff
@@ -0,0 +1,101 @@
+import PropTypes from 'prop-types'
+import InputField from './InputField'
+import Button from './Button'
+import styled from 'styled-components/macro'
+import { useState } from 'react'
+
+Form.propTypes = {
+  onSubmit: PropTypes.func,
+}
+
+export default function Form({ onSubmit }) {
+  const [isDisabled, setIsDisabled] = useState(true)
+
+  return (
+    <StyledForm
+      onSubmit={handleSubmit}
+      onChange={validateForm}
+      aria-label="create a new user"
+      role="form"
+    >
+      <InputField
+        name="name"
+        labelText="name:"
+        placeholder="Please enter your name..."
+        maxLength="40"
+        minLength="5"
+        required
+      />
+      <InputField
+        name="phone"
+        labelText="phone:"
+        placeholder="Please enter your phone number..."
+        minLength="8"
+        required
+      />
+      <InputField
+        name="email"
+        type="email"
+        labelText="email:"
+        placeholder="Please enter your email..."
+        required
+        pattern="^(.+)@(.+)$"
+      />
+      <InputField
+        name="department"
+        labelText="department:"
+        placeholder="Please enter your department..."
+        required
+      />
+      <InputField
+        name="skills"
+        labelText="skills:"
+        placeholder="Please enter your skills..."
```

Comment:
At some point you should point out that these need to be comma separated. 
Also, adding the describing text like `Please enter your skills...` as `placeholder` is not really accessible because some screenreaders do not support placeholder text. Would be better to add it to the label. That's for all your inputs.